Traceback (most recent call last):
  File "/Users/Emman/opt/anaconda3/envs/env-jupyterbook/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/Users/Emman/opt/anaconda3/envs/env-jupyterbook/lib/python3.7/site-packages/nbclient/client.py", line 1082, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/Emman/opt/anaconda3/envs/env-jupyterbook/lib/python3.7/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/Emman/opt/anaconda3/envs/env-jupyterbook/lib/python3.7/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/Users/Emman/opt/anaconda3/envs/env-jupyterbook/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/Users/Emman/opt/anaconda3/envs/env-jupyterbook/lib/python3.7/site-packages/nbclient/client.py", line 536, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/Users/Emman/opt/anaconda3/envs/env-jupyterbook/lib/python3.7/site-packages/nbclient/client.py", line 827, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/Emman/opt/anaconda3/envs/env-jupyterbook/lib/python3.7/site-packages/nbclient/client.py", line 735, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
driver = webdriver.Chrome(chrome_options=option, executable_path=PATH)

keyword = "electrical"

url = "https://www.jobstreet.com.ph/en/job-search/" + keyword + "-jobs/"

driver.get(url)

jobs = driver.find_elements_by_class_name('FYwKg _17IyL_6 _2-ij9_6 _3Vcu7_6 MtsXR_6')

time.sleep(5)
job_list = []

for job in jobs:
    position = job.find_element_by_xpath('//*[@id="jobList"]/div[2]/div[3]/div/div[1]/div/div/article/div/div/div[1]/div[1]/div[2]/h1/a/div').text 
    company = job.find_element_by_xpath('//*[@id="jobList"]/div[2]/div[3]/div/div[1]/div/div/article/div/div/div[1]/div[1]/div[2]/span').text
    location = job.find_element_by_xpath('//*[@id="jobList"]/div[2]/div[3]/div/div[1]/div/div/article/div/div/div[1]/div[1]/span[1]/span').text
    salary = job.find_element_by_xpath('//*[@id="jobList"]/div[2]/div[3]/div/div[1]/div/div/article/div/div/div[1]/div[1]/span[2]').text
    
    job_item = {
        'position': position,
        'company': company,
        'location': location,
        'salary': salary,  
    }

    job_list.append(job_item)

df = pd.DataFrame(job_list)

print(df)
time.sleep(5)

driver.quit()
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
[0;32m<ipython-input-1-8325e8944e57>[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0mdriver[0m [0;34m=[0m [0mwebdriver[0m[0;34m.[0m[0mChrome[0m[0;34m([0m[0mchrome_options[0m[0;34m=[0m[0moption[0m[0;34m,[0m [0mexecutable_path[0m[0;34m=[0m[0mPATH[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      2[0m [0;34m[0m[0m
[1;32m      3[0m [0mkeyword[0m [0;34m=[0m [0;34m"electrical"[0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m [0;34m[0m[0m
[1;32m      5[0m [0murl[0m [0;34m=[0m [0;34m"https://www.jobstreet.com.ph/en/job-search/"[0m [0;34m+[0m [0mkeyword[0m [0;34m+[0m [0;34m"-jobs/"[0m[0;34m[0m[0;34m[0m[0m

[0;31mNameError[0m: name 'webdriver' is not defined
NameError: name 'webdriver' is not defined

